app/frontend/src/views/form/Export.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <BaseSecure :idp="[IDP.IDIR, IDP.BCEIDBUSINESS]">
+    <ExportSubmissions :formId="f" />
+  </BaseSecure>
+</template>
+
+<script>
+import ExportSubmissions from '@/components/forms/ExportSubmissions.vue';
+import { IdentityProviders } from '@/utils/constants';
+export default {
+  name: 'Export',
+  components: {
+    ExportSubmissions,
+  },
+  props: {
+    f: {
```

Comment:
can you change the prop name to be more meaningful, too short

Reply:
Okay. I was following the pattern in the codes